ci/do_ci.sh

```diff
@@ -92,6 +92,22 @@ elif [[ "$1" == "cmake.legacy.test" ]]; then
   make
   make test
   exit 0
+elif [[ "$1" == "cmake.legacy.exporter.otprotocol.test" ]]; then
+  cd "${BUILD_DIR}"
+  rm -rf *
+  export BUILD_ROOT="${BUILD_DIR}"
+  ${SRC_DIR}/tools/build-gtest.sh
+  ${SRC_DIR}/tools/build-benchmark.sh
+  cmake -DCMAKE_BUILD_TYPE=Debug  \
+        -DCMAKE_CXX_STANDARD=11 \
+        -DWITH_OTLP=ON \
+        "${SRC_DIR}"
+  grpc_cpp_plugin=`which grpc_cpp_plugin`
+  proto_make_file="CMakeFiles/opentelemetry_proto.dir/build.make"
+  sed -i "s~gRPC_CPP_PLUGIN_EXECUTABLE-NOTFOUND~$grpc_cpp_plugin~" ${proto_make_file} #fixme
```

Comment:
What is this for?

Reply:
This is actually copied from here (existing otlp build action on ubuntu-latest):
https://github.com/open-telemetry/opentelemetry-cpp/blob/ef16d0048a0ccd6d664b86da1c95294633965bad/ci/do_ci.sh#L103

Somehow, cmake is not able to find the `grpc_cpp_plugin` executable during configuration and so it has to be replaced manually in generated `make` file before doing the actual `make`.
I will debug this separately and fix in another PR.